_stbt/control_gpl.py

```diff
@@ -1,16 +1,68 @@
+from .logging import debug
+
+
 class HdmiCecControl(object):
+    cecconfig = {}
+    lib = {}
```

Comment:
I don't think either of these should be here.  They look like they should be attached to the object rather than the class.  Removing these two lines should be sufficient.